spec/factories/reservations.rb

```diff
@@ -98,7 +98,7 @@
     transient { user nil }
 
     after(:create) do |reservation, evaluator|
-      reservation.order.update_attribute(:user_id, evaluator.user.id) if evaluator.user
+      reservation.order.update_attributes(user_id: evaluator.user.id) if evaluator.user
```

Comment:
Is there ever a case where this could fail validations? Or is there some callback we're relying on? In factories like this, I prefer the validation/callback-free `update_attribute` rather than `update_attributes`.